msal/build.gradle

```diff
@@ -208,6 +211,8 @@ dependencies {
     }
     implementation "cz.msebera.android:httpclient:$rootProject.ext.mseberaApacheHttpClientVersion"
     implementation "androidx.constraintlayout:constraintlayout:$rootProject.ext.constraintLayoutVersion"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$rootProject.ext.kotlinXCoroutinesVersion"
```

Comment:
Please add PR description and in the description also mention why you are introducing new dependencies to MSAL Android.

Reply:
Your PR description says you are only adding new dependencies in tests but that's not what I'm seeing here. Please specify why you need to add this in MSAL.